tools/buildTools/CMakeLists.txt

```
@@ -10,10 +10,13 @@ INSTALL(PROGRAMS "rced.py" DESTINATION /opt/robocomp/bin RENAME rced )
 INSTALL(PROGRAMS "rcbuild.py" DESTINATION /opt/robocomp/bin RENAME rcbuild )
 INSTALL(PROGRAMS "rccomp.py" DESTINATION /opt/robocomp/bin RENAME rccomp )
 INSTALL(PROGRAMS "rccd_utils.py" DESTINATION /opt/robocomp/bin RENAME rccd_utils )
+INSTALL(PROGRAMS "rcfind.py" DESTINATION /opt/robocomp/bin RENAME rcfind )
 INSTALL(PROGRAMS "rccd.sh" DESTINATION /opt/robocomp/share ) #source this in .bashrc
-INSTALL(CODE "message(STATUS \"Installing rccd in ~./bashrc\")
+INSTALL(PROGRAMS "rbcd.sh" DESTINATION /opt/robocomp/share )
+INSTALL(PROGRAMS "rbcd.zsh" DESTINATION /opt/robocomp/share )
+INSTALL(CODE "message(STATUS \"Installing rccd in ~/.bashrc and ~/.zshrc\")
 execute_process(COMMAND sh \"${CMAKE_CURRENT_LIST_DIR}/setup.sh\")")
```

```suggestion
INSTALL(PROGRAMS "rbcd.sh" DESTINATION /opt/robocomp/share )
INSTALL(PROGRAMS "rbcd.zsh" DESTINATION /opt/robocomp/share )
add_custom_target(user_install
    COMMAND sh \"${CMAKE_CURRENT_LIST_DIR}/setup.sh\"
    COMMAND /opt/robocomp/bin/rcworkspace ${RoboComp_SOURCE_DIR}/components/ --accept-all
    COMMENT "Installing rccd in ~/.bashrc and ~/.zshrc"
)
```
what do you think. Would it resolve the root problem? It's another command to be executed on installation, but I don't find another solution... do you?